base/setup/reactos/treelist.c

```diff
@@ -4928,7 +4928,7 @@ static int TreeListSetItem(TreeListData *pData, const TV_ITEM *pItem) {
 		}
 
 		if((uBits & TVIS_EXPANDED) && pEntry->uFirstChild) {	// Sollen Teile auf/zugeklappt werden
-			uMask		   &= TVIS_EXPANDPARTIAL | TVIS_EXPANDPARTIAL;
+			uMask		   &= TVIS_EXPANDED | TVIS_EXPANDPARTIAL;
```

Comment:
This file is not ROS own code...
How do you know this change is correct?

Reply:
Upstream code looks abandonned for years.
Change seems correct as proposed by a smart dev in Jira :-)